src/tasks/installers/operator.ts

```diff
@@ -60,6 +61,66 @@ export class OperatorTasks {
           }
         }
       },
+      {
+        title: 'Checking for pre-created TLS secret',
+        // In case of Openshift infrastructure the certificate from cluster router will be used, so no need in the `che-tls` secret.
+        skip: () => !isKubernetesPlatformFamily(flags.platform),
```

Comment:
Is it better `isOpenshiftplatform` ?

Reply:
To me it will sound like is `platform === 'openshift'`

Reply:
I mean `isOpenshiftPlatformFamily`

Reply:
Yes, exactly. To me, if I rename `isOpenshiftPlatformFamily` to `isOpenshiftPlatform`, it would like the method just checks single value: `platform === 'openshift'`. So, I would prefer to keep the name as is.